server/src/services/notification.service.ts

```diff
@@ -70,6 +83,107 @@ export class NotificationService {
     return JobStatus.SUCCESS;
   }
 
+  async _getAlbumThumbnailAttachment(album: AlbumEntity): Promise<EmailImageAttachment | undefined> {
```

Comment:
get rid of the underscore and just add `private` before the method and move it to the bottom of the file.